vault/resource_auth_backend_test.go

```diff
@@ -55,12 +54,9 @@ func testAccCheckAuthBackendDestroy(s *terraform.State) error {
 func testResourceAuth_initialConfig(path string) string {
 	return fmt.Sprintf(`
 resource "vault_auth_backend" "test" {
-	type = "github"
-	path = "%s"
 	description = "Test auth backend"
-	default_lease_ttl_seconds = 3600
-	max_lease_ttl_seconds = 86400
-	listing_visibility = "unauth"
```

Comment:
These are values that can be set in the tune block, right? Should we set and check them in the test that way, instead of not setting them at all?

Reply:
Think those are covered in the tune specific test. Will double check.

Reply:
While investigating the fix for this issue I hit https://github.com/hashicorp/terraform-plugin-sdk/issues/160 . The TF advised against submitting a fix for the issue, so I am inclined to save this pre-existing wrinkle for another time.